components/textbox/textbox.jsx

```diff
@@ -290,7 +290,6 @@ export default class Textbox extends React.Component {
                 <div
                     ref='preview'
                     className='form-control custom-textarea textbox-preview-area'
-                    style={{display: this.state.preview ? 'block' : 'none'}}
```

Comment:
I don't think this logic makes sense.  This will always render with ``style={{display: 'block'}}`` since it's inside the `if (this.state.preview)` block.  Given that style, it's not affecting the preview rendering.